client/app/hosting/task/hosting-task.controller.js

```diff
@@ -17,8 +17,7 @@ angular.module("App").controller(
                 TODO: "TODO",
                 INIT: "INIT"
             };
-            this.tasksList = null;
-            this.loading = false;
+            this.tasksList = undefined;
```

Comment:
Why would you define to `undefined` by default, this is already the default value

Reply:
I had almost the same argument with another developper on not assigning undefined, this is only for code readability